app/notifiers/publishers/job_application_received_notifier.rb

```diff
@@ -0,0 +1,35 @@
+class Publishers::JobApplicationReceivedNotifier < Noticed::Event
+  deliver_by :database
+  delegate :created_at, to: :record
+  param :vacancy, :job_application
+
+  notification_methods do
```

Comment:
As of v2 we need to wrap these methods in the notifications_method block to make them available on the notification, rather than the event of the same name